shotover-proxy/tests/cassandra_int_tests/mod.rs

```diff
@@ -24,25 +25,27 @@ mod native_types;
 mod prepared_statements;
 #[cfg(feature = "alpha-transforms")]
 mod protect;
+mod schema_awaiter;
 mod table;
 mod udt;
 
-#[test]
+#[tokio::test(flavor = "multi_thread")]
 #[serial]
-fn test_passthrough() {
+async fn test_passthrough() {
     let _compose = DockerCompose::new("example-configs/cassandra-passthrough/docker-compose.yml");
 
     let shotover_manager =
         ShotoverManager::from_topology_file("example-configs/cassandra-passthrough/topology.yaml");
 
     let connection = shotover_manager.cassandra_connection("127.0.0.1", 9042);
+    let schema_awaiter = SchemaAwaiter::new("127.0.0.1:9043").await;
```

Comment:
Is this one meant to connect to a different port? The rest connect directly to Shotover it seems.

Reply:
hmmm they are all meant to connect directly to cassandra

Reply:
I dont see any that connect directly to shotover

Reply:
Oops, I looked at the cluster one and didn't notice the IP was different.